src/store/actionTypes.js

```diff
@@ -34,6 +34,26 @@ export const FETCH_PAYMENTS_REQUEST = 'FETCH_PAYMENTS_REQUEST';
 export const FETCH_PAYMENTS_SUCCESS = 'FETCH_PAYMENTS_SUCCESS';
 export const FETCH_PAYMENTS_FAILURE = 'FETCH_PAYMENTS_FAILURE';
 
+export const FETCH_TEACHERS_REQUEST = 'FETCH_TEACHERS_REQUEST';
+export const FETCH_TEACHERS_SUCCESS = 'FETCH_TEACHERS_SUCCESS';
+export const FETCH_TEACHERS_FAILURE = 'FETCH_TEACHERS_FAILURE';
+
+export const DELETE_TEACHER_REQUEST = 'DELETE_TEACHER_REQUEST';
+export const DELETE_TEACHER_SUCCESS = 'DELETE_TEACHER_SUCCESS';
+export const DELETE_TEACHER_FAILURE = 'DELETE_TEACHER_FAILURE';
+
+export const ADD_TEACHER_REQUEST = 'ADD_TEACHER_REQUEST';
+export const ADD_TEACHER_SUCCESS = 'ADD_TEACHER_SUCCESS';
+export const ADD_TEACHER_FAILURE = 'ADD_TEACHER_FAILURE';
+
+export const GET_TEACHER_BY_ID_REQUEST = 'GET_TEACHER_BY_ID_REQUEST';
+export const GET_TEACHER_BY_ID_SUCCESS = 'GET_TEACHER_BY_ID_SUCCESS';
+export const GET_TEACHER_BY_ID_FAILURE = 'GET_TEACHER_BY_ID_FAILURE';
+
+export const EDIT_TEACHER_REQUEST = 'EDIT_TEACHER_REQUEST';
+export const EDIT_TEACHER_SUCCESS = 'EDIT_TEACHER_SUCCESS';
+export const EDIT_TEACHER_FAILURE = 'EDIT_TEACHER_FAILURE';
```

Comment:
EDIT заменить на  UPDATE, как в STUDENTS 
```suggestion
export const EDIT_TEACHER_REQUEST = 'EDIT_TEACHER_REQUEST';
export const EDIT_TEACHER_SUCCESS = 'EDIT_TEACHER_SUCCESS';
export const EDIT_TEACHER_FAILURE = 'EDIT_TEACHER_FAILURE';
```

Reply:
исправлен